jax_rocm_plugin/pjrt/python/__init__.py

```
@@ -140,8 +190,7 @@ def initialize():
         logger.warning("rocm_plugin_extension not found")
         return
 
-    device_count = rocm_plugin_extension.get_device_count()
-    if device_count <= 0:
+    if not is_amd_gpu_available():
         raise ValueError("No GPUs found")
```

Do we know why do we raise here instead of loging + return as above?